examples/cookies_load.rs

```diff
@@ -0,0 +1,35 @@
+use reqwest::header::{HeaderMap, HeaderValue};
+use std::io::{self, Write};
+pub use vrchatapi::apis;
+use vrchatapi::models::{EitherUserOrTwoFactor, TwoFactorAuthCode, TwoFactorEmailCode};
+
+#[tokio::main]
+async fn main() {
+    let mut config = apis::configuration::Configuration::default();
+    config.basic_auth = Some((String::from("username"), Some(String::from("password"))));
+    config.user_agent = Some(String::from("ProjectName/0.0.1 email@example.com"));
+
+    let mut header_map = HeaderMap::new();
+    header_map.append(
+        "Cookie",
+        HeaderValue::from_str(
+            &"auth=[AUTH_COOKIE_HERE]; twoFactorAuth=[TWO_FACTOR_AUTH_COOKIE_HERE]",
+        )
+        .unwrap(),
+    );
+
+    config.client = reqwest::Client::builder()
+        .cookie_store(true)
+        .default_headers(header_map)
+        .build()
+        .unwrap();
```

Comment:
The better way to do this would be to create a cookie jar (similarly to cookie_store), and insert the required values before setting it as a `cookie_provider`.

Reply:
Here is what I'd do:
```
        let mut jar = reqwest::cookie::Jar::default();
        use std::str::FromStr;
        jar.set_cookies(&mut [HeaderValue::from_str(
            &"auth=[AUTH_COOKIE_HERE]; twoFactorAuth=[TWO_FACTOR_AUTH_COOKIE_HERE]",
        )], &url::Url::from_str("https://api.vrchat.cloud").expect("Url not okay"));
        let jar = Arc::new(jar);
        let client = reqwest::Client::builder()
            .cookie_provider(jar)
            .build()?;
```

This way the examples for loading and saving cookies could even be combined.